### ISSUE TITLE
Issue downloading Google-native files

### ISSUE THREAD

**Issue by luke-c-sargent** [rkhwaja/fs.googledrivefs#62]
Attempts to download Google native files (E.g., docs, sheets, slides) produce an error [when using get_media()](https://github.com/rkhwaja/fs.googledrivefs/blob/9568eb49a9084d4d9751eb27fb558ee77558cbfc/fs/googledrivefs/googledrivefs.py#L427):
```
Only files with binary content can be downloaded. Use Export with Google Docs files.
```
a potential solution could be to get the file's metadata type to determine the method of export, then use [export_media()](https://developers.google.com/resources/api-libraries/documentation/drive/v2/python/latest/drive_v2.files.html#export_media) -- though this has a limit of files up to 10MB.

here's [some more info from a stackoverflow article](https://stackoverflow.com/questions/59212443/google-drive-api-with-python-not-allowing-file-download-despite-correct-scopes-b)

thanks!

**Comment by rkhwaja** [comment]
The download method signature allows a dictionary of options. We can add some kind of "export as mime type" option and then try to export as that type.
We can test creating a spreadsheet, doc, presentation, whatever and export it to various formats. The test needs to be able to create the doc/whatever in the test directory which means calling the raw google API or allowing it through googledrivefs somehow.

**Comment by rkhwaja** [comment]
Fixed in v2.3.0
